lighthouse/lighthouse-auth.mjs

```diff
@@ -54,31 +58,15 @@ async function login(page, origin) {
   }
 }
 
-export async function loadInitialPage(page, origin, timeout = 60_000) {
-  let originResponse = await page.goto(origin);
-
-  if (pageFound(originResponse)) {
-    return Promise.resolve(originResponse);
+export async function checkForInitialLoad(page, tries = 0, timeout = 60_000) {
```

Comment:
Should we switch these args since `timeout` is something that might be customized (like in tests), but we always want `tries` to start at 0?